bin/ch_misc.c

```diff
@@ -0,0 +1,190 @@
+/* Copyright © Triad National Security, LLC, and others. */
+
+#define _GNU_SOURCE
+#include <libgen.h>
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/mount.h>
+#include <sys/stat.h>
+#include <sys/statvfs.h>
+#include <unistd.h>
+
+#include "config.h"
+#include "ch_misc.h"
+
+
+/** Macros **/
+
+/* Number of supplemental GIDs we can deal with. */
+#define SUPP_GIDS_MAX 128
+
+
+/** Constants **/
+
+/* Names of verbosity levels. */
+const char *VERBOSE_LEVELS[] = { "error", "warning", "info", "debug" };
+
+
+/** External variables **/
+
+/* Level of chatter on stderr desired (0-3). */
+int verbose;
+
+
+/** Function prototypes (private) **/
+
+// none
```

Comment:
Is this section just here to provide symmetry with ch_core.h?

Reply:
Yes.